src/main/org/apache/tools/ant/taskdefs/XmlProperty.java

```diff
@@ -404,17 +404,14 @@ public Object processNode(Node node, String prefix, Object container) {
         }
         String nodeText = null;
         boolean emptyNode = false;
-        boolean semanticEmptyOverride = false;
-        if (node.getNodeType() == Node.ELEMENT_NODE
+        boolean semanticEmptyOverride = node.getNodeType() == Node.ELEMENT_NODE
                 && semanticAttributes
                 && node.hasAttributes()
                 && (node.getAttributes().getNamedItem(VALUE) != null
-                        || node.getAttributes().getNamedItem(LOCATION) != null
-                        || node.getAttributes().getNamedItem(REF_ID) != null
-                        || node.getAttributes().getNamedItem(PATH) != null || node.getAttributes()
-                        .getNamedItem(PATHID) != null)) {
-            semanticEmptyOverride = true;
-        }
+                || node.getAttributes().getNamedItem(LOCATION) != null
```

Comment:
please revert the whitespace change @arturobernalg

Reply:
hi @bodewig 
Changed.
TY